src/App.tsx

```diff
@@ -19,7 +20,12 @@ export const theme: typeof MD3DarkTheme = {
 
 export default function App() {
   useEffect(() => {
-    void DwnService.initSqliteDwn();
+    async function init() {
+      void DwnService.initMemoryDwn();
+      const { web5, did: myDid } = await Web5.connect();
```

Comment:
I believe that `Web5.connect()` won't be called from the identity agents like the wallet. This concept is going through a pretty major refactor currently by @frankhinek on the web5-js side.

Would like to hold  off on merging this in until we have a solid understanding of what this re-write entails for clients. @frankhinek if you have any insights here, would greatly appreciate it!